"""
Пример простой формы на Qt с обработчиком
"""
import sys
from PyQt5 import QtWidgets
from day3 import design

class ExampleApp(QtWidgets.QMainWindow, design.Ui_MainWindow):
    def __init__(self):
        super().__init__()
        self.setupUi(self)
        self.build_handlers()

    def build_handlers(self):
        self.pushButton.clicked.connect(self.on_button_click)

    def on_button_click(self, event):
        message = self.lineEdit.text()
        self.plainTextEdit.appendPlainText(message)
        self.lineEdit.setText('')

def main():
    app = QtWidgets.QApplication(sys.argv)
    window = ExampleApp()
    window.show()
    app.exec_()

if __name__ == '__main__':
    main()
